Firmware/RTK_Surveyor/FileSdFatMMC.h

```diff
@@ -195,21 +214,21 @@ class FileSdFatMMC : public SdFile
 #endif
 };
 
-//Update the file access and write time with date and time obtained from GNSS
-// These are SdFile-specific. SD_MMC does this automatically
+// Update the file access and write time with date and time obtained from GNSS
```

Comment:
If you start pulling on the thread that is comment spacing, you'll end up naked! In all seriousness, I don't care either way, but I do care about your time. I pretty much can't stop doing //Text do we'll just have to agree to ignore each other's comments.